### DISCUSSION TITLE
Be able to not install extra extensions that are not needed? search engines

### DISCUSSION THREAD

**Discussion by BarbzYHOOL** [Floorp-Projects/Floorp#502] [Feature Request]
I went to about:support and saw we have these extensions:
Yahoo! JAPAN	extension	1.0	true	yahoo-jp@search.mozilla.org
You.com	extension	1.0	true	you.com@search.mozilla.org
Bing

and a few more search engines

They seem pretty interesting, I didn't know about you.com, and searxng might be useful, but Yahoo Japan is useless for non japanese, especially because of this "2022年4月6日 (水) よりYahoo! JAPANは欧州経済領域（EEA）およびイギリスからご利用いただけなくなりました" (i let you translate it :D)

**Comment by Ryosuke-Asano** [comment]
Oh. I see.

I will remove Yahoo

**Comment by Ryosuke-Asano** [comment]
Yahoo Japan will be removed at 11.5.0.
